src/apps/allbridge/api.ts

```diff
@@ -0,0 +1,95 @@
+import got from 'got'
+import { NetworkId } from '../../types/networkId'
+import { Address } from 'viem'
+
+type SupportedAllBridgeChainSymbols =
+  | 'ETH'
+  | 'CEL'
+  | 'POL'
+  | 'ARB'
+  | 'OPT'
+  | 'BAS'
+
+const ALLBRIDGE_BLOCKCHAIN_SYMBOL_TO_NETWORK_ID: Record<
+  SupportedAllBridgeChainSymbols,
+  NetworkId
+> = {
+  ETH: NetworkId['ethereum-mainnet'],
+  CEL: NetworkId['celo-mainnet'],
+  POL: NetworkId['polygon-pos-mainnet'],
+  ARB: NetworkId['arbitrum-one'],
+  OPT: NetworkId['op-mainnet'],
+  BAS: NetworkId['base-mainnet'],
+}
+
+type AllBridgeApiResponse = {
```

Comment:
is this documented somewhere or did we just make it ourselves from the response, if the former could we link to it?

Reply:
I believe that @MuckT just made it from the response, I copied it over from his PR where he didn't end up needing it.

Reply:
Used the response from https://core.api.allbridgecoreapi.net/token-info and set the types to match.